docs/source/routing/get-started.mdx

```diff
@@ -0,0 +1,293 @@
+---
+title: Apollo Router Quickstart
+subtitle: Run a router binary
+description: This quickstart tutorial walks you through installing an Apollo Router binary, running it with an example supergraph schema and a YAML configuration file, and making test queries with Apollo Sandbox.
+---
+
+import ElasticNotice from "../../shared/elastic-notice.mdx";
+
+Hello! Let's run Apollo Router for the first time, using the simple scenario of developing and deploying locally.
+
+In this guide, you will:
+
+- Download and install a router binary.
+- Download a supergraph schema.
+- Create a router YAML configuration file.
+- Run the router in development mode.
+- Make a query to the running router.
+
+## 1. Install router binary
+
+Let's start by downloading and installing a router binary locally.
+
+1. Download and install the latest version of the router binary with a single command line:
+
+    ```bash showLineNumbers=false
+    curl -sSL https://router.apollo.dev/download/nix/latest | sh
+    ```
+
+    <ExpansionPanel title="Download a specific router version">
+
+    To download and install a specific version of router, set the version in the download URL path.
+
+    For example, to download router v2.0.0:
+
+    ```bash showLineNumbers=false
+    curl -sSL https://router.apollo.dev/download/nix/v2.0.0 | sh
+    ```
+
+    </ExpansionPanel>
+
+    Optionally, go to [router releases](https://github.com/apollographql/router/releases) in GitHub to download and extract a bundle.
+
+1. Check that your router installed successfully by running the `router` binary from your project's root directory:
+
+    ```bash showLineNumbers=false
+    ./router --version
+    ```
+
+
+## 2. Download supergraph schema
+
+A router needs a schema for the federated graph, or _supergraph_, that it's orchestrating. This guide uses an example supergraph schema, which you download and provide to the router.
```

Comment:
Can we rephrase this to something more like this? I think this needs more context about the 3 config pieces that Router needs to start up, and we need to describe why the supergraph schema is important and what exactly it does for folks.

Router needs 3 pieces of configuration to start up:
1. supergraph schema – the supergraph schema tells router _everything_ it needs to know to effectively route your queries. this is a graphql schema annotated with federation directives
2. router config – this yaml file tells router what it needs to know about header propagation, error handling, what subgraphs to route to, etc.
3. entitlement – this tells router what plan you're on and therefore what features you have access to